collect_app/src/main/java/org/odk/collect/android/adapters/model/Topic.java

```diff
@@ -0,0 +1,59 @@
+package org.odk.collect.android.adapters.model;
+
+public class Topic {
+    private String id;
+    private String title;
+    private String icon;
+    private int discussionCount;
+
+    public Topic() {
+
+    }
+
+    public Topic(String id, String title, String icon, int discussionCount) {
+        this.id = id;
+        this.title = title;
+        this.icon = icon;
+        this.discussionCount = discussionCount;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getIcon() {
+        return icon;
+    }
+
+    public void setIcon(String icon) {
+        this.icon = icon;
+    }
+
+    public int getDiscussionCount() {
+        return discussionCount;
+    }
+
+    public void setDiscussionCount(int discussionCount) {
+        this.discussionCount = discussionCount;
+    }
+
+    public void incrementDiscussionCount() {
+        discussionCount++;
```

Comment:
cette méthode semble n'avoir aucun effect. A moins que la syntax soit  this.discussionCount. 

Mais pourquoi le nombre de discussion d'un topic devrait être modifé par la class Topic ?

Reply:
C'est le nombre de discussion dans le Topic et elle est incrementer a chaaque fois qu'une discussion est creer dans le Topic